Mage/src/main/java/mage/abilities/effects/common/CreateTokenTargetEffect.java

```diff
@@ -69,6 +69,11 @@ public String getText(Mode mode) {
             return staticText;
         }
         StringBuilder sb = new StringBuilder();
+
+        if (mode.getTargets().isEmpty()) {
+            return "";
```

Comment:
Effect must return real text, not empty string. You can add "target ", "target player " or other text.

Reply:
Addressed